lib/repl.ts

```diff
@@ -8,19 +8,16 @@
  * @license MIT
  */
 
-'use strict';
-
-const fs = require('fs');
-const path = require('path');
-const net = require('net');
-const repl = require('repl');
+import { chmodSync, readdirSync, statSync, unlink, unlinkSync } from 'fs';
+import * as net from 'net';
+import * as path from 'path';
+import * as repl from 'repl';
 
 const Repl = {
```

Comment:
Apparently how I'd do this now is `export const Repl = new class {`

Reply:
Which allows members to have types:

```
socketPathnames: Set<string> = new Set();
```

Reply:
I dig.